### PR TITLE
Issue 46

### PR DESCRIPTION
**Pull request by NgGeorge**
Circumvented change detection by concatenating the model data with the new array of invited emails, then setting it directly.

### CODE REVIEW

File: server/app/controllers/meeting.js [hunk 1]
```diff
@@ -121,7 +124,8 @@ router.post('/invite', function(req, res, next) {
         responseLink: 'http://ezpick.herokuapp.com/meeting/' + meeting.id,
       }
     );
-
+    newEmails = meeting.invited.concat(newEmails);
```
Comment (zmbc) [inline server/app/controllers/meeting.js]: On this line, `newEmails` stops being the new emails, which is confusing. You can simply do `meeting.invited = meeting.invited.concat(newEmails)`

File: server/app/controllers/meeting.js [hunk 2]
```diff
@@ -106,9 +108,10 @@ router.post('/invite', function(req, res, next) {
       id: req.body.id,
     },
   }).then(function(meeting) {
+    var newEmails = new Array();
```
Comment (zmbc) [inline server/app/controllers/meeting.js]: Prefer `[]` syntax

File: server/app/controllers/meeting.js [hunk 3]
```diff
@@ -44,8 +44,10 @@ router.get('/:id', function (req, res, next) {
     });
   }).catch(function(err) {
     res.status(404).json({
-      success: false
+      success: false,
+      error: err
     });
+    console.log(error);
```
Comment (zmbc) [inline server/app/controllers/meeting.js:50]: Remove the changes to this method, not related to the issue

**Review by zmbc — CHANGES_REQUESTED** [review]

File: server/app/controllers/meeting.js (hunk 1 above)
Comment (NgGeorge) [reply server/app/controllers/meeting.js]: Great call, I actually did something similar but it broke so I just swapped it into that. Don't know what I was thinking.

**Review by zmbc — APPROVED** [review]

**Pull request merged**